testsuite/tests/lib-unix/common/uexit.ml

```diff
@@ -0,0 +1,11 @@
+(* TEST
+* hasunix
+include unix
+** bytecode
+** native
+*)
+
+let _ =
+  at_exit (fun () -> print_string "B\n"; flush stdout);
+  print_string "A\n"; (* don't flush *)
```

Comment:
I thought that the channel I/O doesn't guarantee _to flush_ without an explicit instruction, but isn't it (or a future implementation) at liberty to flush _if it wants to_? Isn't the test sufficient without this line anyhow?

Reply:
Yes, this test relies on a particular implementation of I/O channels w.r.t. flushing, but said behavior hasn't changed since Caml Light, IIRC, so the test should be OK.  I'll fix it later if it causes problems.